assets/css/settings.scss

```diff
@@ -656,3 +656,36 @@
 		}
 	}
 }
+
+.sensei-learners-wrap .search-box {
```

Comment:
Could we keep this code close to the other CSS for "Learner Management" for easier maintainability? We should probably also probably rename the "Learner Management" comment to "Students" in this file.